<?xml version="1.0" ?>
<launch>
<arg name="joy_dev" default="/dev/input/js0" />
    <arg name="cmd_topic" default="cmd_js" />

    <node pkg="joy" type="joy_node" name="joy_node">
        <param name="dev" value="$(arg joy_dev)" />
        <param name="deadzone" value="0.0" />
    </node>

    <node pkg="robotic_ultrasound" type="joystick_to_franka.py" name="franka_teleop" output="screen">
        <remap from="cmd_js" to="$(arg cmd_topic)" />
        <param name="ax0deadzone" value="0.05" />
        <param name="ax1deadzone" value="0.05" />
        <param name="ax4deadzone" value="0.08ss" />
    </node>
<node pkg="robotic_ultrasound" type="LUS_auto_landing_manual.py" name="LUS_auto_landing_manual" output="screen" />
</launch>